Project-1/README.Giacomo.md

```diff
@@ -0,0 +1,322 @@
+  The second formula reads "for all xs, there is a y, such that for all zs". Any relation coming afterwards will relate all the element x of the domain to at least an element y in the domain, which in turn is related to all elements z in the domain. 
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  The first formula reads "for all xs, there is a y, such that for all zs, there is a w such that"
+  Any relation coming afterwards will relate all elements x in the domain with at least an element y, which in turn relates to all the elements z in the domain, which in turn are each related to at least one element w.
+``The second formula reads "there is an x, such that for all ys, there is a z, such that all w".
+  Any relation coming afterwards will relate one element x in the domain with all elements y, which are each related to an element y, which in turn relate to all the elements w in the domain.
+
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+<img width="205" alt="7" src="https://user-images.githubusercontent.com/117365623/218544552-d4b55a2f-6529-4459-a7f1-8949af713f85.png">
+
+```
+ {(a,b)(b,a)(a,a)} 
+ The second statement is true in this model. There is an x that relates every y (a, related to a and b) There is an x that relates to a z (where a is x and b is z), all ys (a and b) are related to some z (a is related to a and b, b is related to a)
+ The first statement is not true. It is not true that all xs (a and b) are related to all zs (b is not related to a)
+A graph is provided in the attached file.
+
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+  ```
+See attached images.
+
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+  <img width="292" alt="8-a" src="https://user-images.githubusercontent.com/117365623/218544428-355ea0a8-b36a-4be7-aadb-7f4d47c89b10.png">
+
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+  <img width="289" alt="8-b" src="https://user-images.githubusercontent.com/117365623/218544451-7c56e309-7114-4dd8-b0ea-75b964f764c1.png">
+
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  
+  <img width="289" alt="8-c" src="https://user-images.githubusercontent.com/117365623/218544502-17e98b99-5a7a-4692-9ca8-72e714aad700.png">
+
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+  <img width="301" alt="8-d" src="https://user-images.githubusercontent.com/117365623/218544517-8cac0986-7773-4451-bfd5-74b6a94b3e2d.png">
+
+```
+	
+9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+
+Using http://teachinglogic.liglab.fr/DN/index.php, since I couldn't make the other website work.
+
+a) 
+assume -(Px & Qx).

```

Comment:
You have not instantiated the universal to a constant. 

You have not included any justification for inferences in your proofs...